config.yaml

```diff
@@ -86,6 +86,17 @@ buffer:
   # - Focused: Only show input when the buffer is focused
   input_visibility: Always
 
+  # Topic banner settings:
+  topic_banner:
+    # Show topic messages in a topic banner at the topic of a pane instead
+    # of server messages.  Topic changes will still show up as a server
+    # message.
+    # - Default is false.
+    enabled: false
```

Comment:
The more descriptive naming is a good idea.  Thanks & implemented :)  Wound up making the max height a struct field since I couldn't work out how to give it a default value otherwise.